src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs

```diff
@@ -1,8 +1,7 @@
-namespace NServiceBus.Sagas
+namespace NServiceBus.Saga
```

Comment:
> If we do we'll have both a class and a namespace with the same name ==
pain?

There is no pain, as I said before that is how it is currently out in prod.

On Friday, March 27, 2015, Andreas Öhlund <notifications@github.com> wrote:

> In src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs
> <https://github.com/Particular/NServiceBus/pull/2693#discussion_r27279405>
> :
>
> > @@ -1,8 +1,7 @@
> > -namespace NServiceBus.Sagas
> > +namespace NServiceBus.Saga
>
> I propose we move the saga base class to the root `NServiceBus` namespace
> to be more discoverable OOTB. If we do that why should we keep the
> `NServiceBus.Saga` namespace? If we do we'll have both a class and a
> namespace with the same name == pain?
> … <#14c5a26a2031e22c_>
> On Fri, Mar 27, 2015 at 8:28 AM, John Simons <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote: In
> src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs <
> https://github.com/Particular/NServiceBus/pull/2693#discussion_r27279285>
> : > @@ -1,8 +1,7 @@ > -namespace NServiceBus.Sagas > +namespace
> NServiceBus.Saga Not worth the effort IMO … <#14c5a219cec958a8_> On Friday,
> March 27, 2015, John Simons <john.simons@particular.net
> <javascript:_e(%7B%7D,'cvml','john.simons@particular.net');>> wrote: > I
> agree but all public classes are currently siting in Saga so moving all >
> those to sagas is a much bigger change > > On Friday, March 27, 2015,
> Andreas Öhlund <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');> >
> <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>> wrote: >
> In src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs > <
> https://github.com/Particular/NServiceBus/pull/2693#discussion_r27279193>
> > : > > > @@ -1,8 +1,7 @@ > > -namespace NServiceBus.Sagas > > +namespace
> NServiceBus.Saga > > Makes alot of sense. But I think Sagas is a better
> namespace name? > > — > Reply to this email directly or view it on GitHub >
> <https://github.com/Particular/NServiceBus/pull/2693/files#r27279193>. >
> — Reply to this email directly or view it on GitHub <
> https://github.com/Particular/NServiceBus/pull/2693/files#r27279285>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/2693/files#r27279405>.
>